pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -45,6 +45,26 @@ self: super: {
   # Needs older QuickCheck version
   attoparsec-varword = dontCheck super.attoparsec-varword;
 
+  # Fixed at head, but hasn't cut a release in awhile.
```

Comment:
Please create an issue upstream asking for a new release to hackage, and link the issue in a comment here so we have a way to track when we can remove these overrides.